keyboards/handwired/selene/keymaps/Bpendragon/keymap.c

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2020 Bpendragon
```

Comment:
Please rename this keymap to `default`, or add a separate default and rename this one to lowercase.

Reply:
Default added in 569e368 `bpendragon` forced lowercase in a6187a9